pom.xml

```diff
@@ -1,26 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <!-- Baseline Jenkins version you use to build and test the plugin. Users must have this version or newer to run. -->
-    <version>4.66</version>
+    <version>5.12</version>
     <relativePath />
   </parent>
-  <groupId>org.jenkins-ci.plugins</groupId>
+
   <artifactId>cors-filter</artifactId>
-  <version>1.2-SNAPSHOT</version>
+  <version>${changelist}</version>
+  <groupId>org.jenkins-ci.plugins</groupId>
   <packaging>hpi</packaging>
-
   <name>CORS support for Jenkins</name>
   <description>Support Cross Site Posts to access Jenkins resources</description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Cors+Filter+Plugin</url>
```

Comment:
you'll want to change this to point to your GitHub repo in order for docs to display from your github repo instead of the old wiki (that is in read-only mode)

Reply:
@timja done